rules_gapic/java/java_gapic_pkg_deps_resolution.bzl

```diff
@@ -197,6 +197,8 @@ def _possibly_artifact_id_version_chunk(chunk):
     return True
 
 def _possibly_artifact_id_classifier_chunk(chunk):
+    if chunk == "android" or chunk == "jre":
```

Comment:
Can you comment on why this is necessary?

Reply:
Added a comment